### DISCUSSION TITLE
all setting are reset after restarting the OBS.

### DISCUSSION THREAD

**Discussion by WaveSimm** [sorayuki/obs-multi-rtmp#75] [Q&A]
Hi,
It is a great plugin for broadcasting.
However, I am struggling to set the plugin so far since the sets are reset after restarting OBS.
I have used the latest version which is now an install file and didn't change the default folder.
What did I wrong to install or set the plugin?

**Comment by sorayuki** [comment]
Hi, you have installed it without anything wrong if it works.
I hVe heard several cases about setting lost. But I have no idea why it happened.
I judt used obs's config_t class to handle the setting's saving and loading. It should be saved in obs's ini file with base64 encoded...
Maybe I can have a try to put them into a standalone json file.

**Reply by ZukoHama** [reply]
como puedo instalar este plugin en ubuntu

**Comment by ZBlackw1** [comment]
The plugin would lose all of the settings I put in until I actually went live using those settings. After that, all of the destinations I had configured stayed in place. Maybe that will work for your issue @WaveSimm.

**Reply by WaveSimm** [reply]
WoW.. Thanks for your reply.. Yes. the setting can be saved after streaming using this plugin!!!

@sorayuki  I think you need to command this issue.

**Reply by sorayuki** [reply]
I can't reproduce the problem.
Is it possible to provide a screen recording showing how it happened?

**Comment by VilaxCam** [comment]
I have the same problem, when i re-opening my OBS rtm plugin gone.

https://user-images.githubusercontent.com/83318426/116325659-51f2d300-a7cb-11eb-9fcb-1ec6076a9ace.mp4

**Comment by frbelotto** [comment]
I have the same issue here. The settings are reset every time I open OBS. I will try later to start a streaming after setting up it to see if solves the issue.

**Comment by sorayuki** [comment]
Hi, it may have been fixed in 0.2.6.1.

**Reply by WaveSimm** [reply]
great !!!
